graylog2-web-interface/src/components/common/SearchForm.jsx

```diff
@@ -162,6 +171,7 @@ class SearchForm extends React.Component {
           <div className="form-group has-feedback">
             {this.props.label && <label htmlFor="common-search-form-query-input" className="control-label">{this.props.label}</label>}
             <input id="common-search-form-query-input"
+                   ref={(ref) => { this.searchInput = ref; }}
```

Comment:
What's your feeling on using React's [`createRef`](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs)?